{"name": "Проверка ошибки при авторизации с неверным логином или паролем", "status": "failed", "statusDetails": {"message": "AssertionError: assert (404 == 404 and '{\"code\":404,... не найдена\"}' == '{\"message\": ... не найдена\"}'\n +  where 404 = <Response [404]>.status_code\n  - {\"message\": \"Учетная запись не найдена\"}\n  ?            -\n  + {\"code\":404,\"message\":\"Учетная запись не найдена\"}\n  ?  +++++++++++)", "trace": "self = <test_login_courier.TestLoginCourier object at 0x000001919AC29850>\nregistered_courier = {'login': 'blngnszino', 'password': 'untvcbesyt'}\nfield = 'login'\n\n    @allure.title('Проверка ошибки при авторизации с неверным логином или паролем')\n    @allure.description(\n        'Отправляем запрос, который авторизует курьера, с неверным логином или паролем и проверяем, '\n        'что вернулись ожидаемые код и текст ответа об ошибке'\n    )\n    @pytest.mark.parametrize('field', [\"login\", \"password\"])\n    def test_login_courier_with_invalid_log_or_pass(self, registered_courier, field):\n        payload = registered_courier.copy()\n        payload[field] += 'invalid'\n        response = requests.post('https://qa-scooter.praktikum-services.ru/api/v1/courier/login', data=payload)\n>       assert response.status_code == 404 and response.text == '{\"message\": \"Учетная запись не найдена\"}'\nE       assert (404 == 404 and '{\"code\":404,... не найдена\"}' == '{\"message\": ... не найдена\"}'\nE        +  where 404 = <Response [404]>.status_code\nE         - {\"message\": \"Учетная запись не найдена\"}\nE         ?            -\nE         + {\"code\":404,\"message\":\"Учетная запись не найдена\"}\nE         ?  +++++++++++)\n\ntests\\test_login_courier.py:29: AssertionError"}, "description": "Отправляем запрос, который авторизует курьера, с неверным логином или паролем и проверяем, что вернулись ожидаемые код и текст ответа об ошибке", "parameters": [{"name": "field", "value": "'login'"}], "start": 1699213339084, "stop": 1699213339265, "uuid": "280195ce-4cef-461f-baad-a01299ee523f", "historyId": "04dd2da92020fbdd24bf7080fcb4e6e2", "testCaseId": "0bcf38a682a08e2b1db1e00f1a2d4d4d", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_courier_with_invalid_log_or_pass", "labels": [{"name": "feature", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-SK1TUPO"}, {"name": "thread", "value": "1872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}